src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/dataBrokerProfiles.module.scss

```diff
@@ -0,0 +1,188 @@
+@import "../../../../../../../tokens";
+
+.contentWrapper {
+  max-width: $content-md;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.buttonsWrapper {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+
+  a {
+    display: block;
+  }
+
+  gap: $spacing-md;
```

Comment:
nit: move `gap` up around L10-12?

Reply:
Weird. I figure linting would've burped on that. Great catch! 

Revised in 67ecbc1513e9465bc8032d4293a452706739913b